core/src/docs/rfcs/1735_operation_extension.md

```diff
@@ -0,0 +1,117 @@
+- Proposal Name: `operation_extension`
```

Comment:
This may not be our final extension to operation. How about 'header_extensions_and_overrides'?

Reply:
Take it easy, we will come up with better names in the future.